node/cmd/guardiand/node.go

```diff
@@ -639,6 +659,9 @@ func runNode(cmd *cobra.Command, args []string) {
 		if *arbitrumContract != "" && !*unsafeDevMode {
 			logger.Fatal("Please do not specify --arbitrumContract")
 		}
+		if *aptosRPC != "" && !*unsafeDevMode {
```

Comment:
Sorry, I think I missed this the first time. Doesn't this mean you won't be able to enable it in mainnet?

Reply:
Fixed.. amusingly, @kcsongor and I looked at this and confused ourselves on if we had to change it